src/components/simulation/SimulationPanel.svelte

```diff
@@ -348,11 +385,22 @@
 
     <fieldset>
       <Collapse title="Simulation History" padContent={false}>
+        <svelte:fragment slot="right">
+          {#if $planSnapshot}
+            <FilterToggleButton
+              label="Simulation"
+              offTooltipContent="Filter simulations by selected snapshot"
+              onTooltipContent="Remove filter"
+              isOn={isFilteredBySnapshot}
```

Comment:
Reverts back to true if you turn it off and then wait a second while a simulation is running